src/main/java/com/kjetland/dropwizard/activemq/ActiveMQConfig.java

```diff
@@ -11,6 +11,9 @@
     @NotNull
     public String brokerUrl;
 
+    @JsonProperty
+    public String healthCheckAppendToBrokerUrl;

```

Comment:
Would be nice to add this parameter to the[ Readme](https://github.com/novamediasverige/dropwizard-activemq-bundle/blob/develop/README.md) file (same comment for healthCheckRequired, even though it was not part of this PR). 
Is there a way to set a default for this value?

Reply:
unfortunately no default value since it's dependent on whether other options are used or not in the brokerUrl. We couldn't find a way to set it in java without parsing brokerUrl either :/

Reply:
Fixed readme and some comments in code